src/pages/[platform]/build-a-backend/add-aws-services/predictions/set-up-predictions/index.mdx

```diff
@@ -86,12 +86,15 @@ const backend = defineBackend({
   auth,
 });
 
+
+// Configure a policy for the required use case.
+// The actions included below cover all supported ML capabilities
```

Comment:
I think this is too hidden, I think we should use a Callout and direct customers to the policy documentations. 

e.g.

"With the following we are adding the policies for all the usecases in this documentation. There is a high chance that you won't need all of these policies. Check the docs .... to select the correct policy"

Reply:
was wondering if we could either add a comment before each group of policies with something like `permissions to enable translate capabilities` or split the policies to their individual docs for readability

Reply:
I am okay with whatever way can teach customers about the policies and limit the usage of it to the only ones that they need.

Reply:
Updated to add a callout, I kept the comment inside the code from more visibility, didn't spilt it up for readability